ElrondNetwork/api.elrond.com
# Transactions endpoint with multiple receivers

## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- No possibility to search transactions for multiple receivers in the same api call
  
## Proposed Changes
- Add receivers parameter as comma-separated array in transaction filter
- Adjust elastic query to search for multiple receivers
- Keep receiver parameter for backwards compatibility (it will be added in receivers array when search in elastic)

## How to test
- Check if any filter with same value for parameters receiver/receivers return same results
- Check for multiple receivers transactions 
- Check if receivers apply with different filters (sender, date range, token, etc)
